### PR TITLE
[Windows] Add libpxc into DEPS.xwalk for RealSense camera support

### PR DESCRIPTION
**Pull request by huningxin**
BUG=XWALK-6028

### CODE REVIEW

**Comment by huningxin** [comment]
https://github.com/crosswalk-project/chromium-crosswalk/pull/305 depends on this PR to build on Windows.

**Comment by huningxin** [comment]
@darktears @halton , PTAL. Thanks!

**Comment by halton** [comment]
LGTM

**Comment by darktears** [comment]
LGTM

**Comment by huningxin** [comment]
Thanks for review! @darktears @halton . Could you please help merge it?

File: DEPS.xwalk [hunk 1]
```diff
@@ -47,6 +47,11 @@ solutions = [
       'src/third_party/khronos/CL':
         crosswalk_git + '/khronos-cl-api-1.2.git@6f4be98d10f03ce2b12c769cd9835c73a441c00f',
 
+      # Include Intel RSSDK headers and library loader for RealSense camera
+      # support.
+      'src/third_party/libpxc':
+        crosswalk_git + '/libpxc.git@WM5_1',
```
Comment (rakuco) [inline DEPS.xwalk]: I don't think there's a downside to using a tag here, but you might want to consider using a full hash just to follow what the other entries (including `DEPS` upstream) do.

**Comment by huningxin** [comment]
@rakuco , updated the PR by using the hash. Please take another look. Thanks!

**Comment by rakuco** [comment]
lgtm, I'd like to land the change to chromium-crosswalk's `.gitignore` first though, otherwise the `src/third_party/libpxc` will be erased and re-checked out on every call to `gclient sync`.

**Comment by huningxin** [comment]
@rakuco , added commits to roll `DEPS.xwalk` and set `use_rssdk=1` by default for Windows. Please take another look. Thanks!

**Comment by halton** [comment]
This PR contains three commits, from my point-of-view, it is okay and LGTM. Unless others think we should follow one PR one commit policy.

**Comment by rakuco** [comment]
Perfect, lgtm!

**Pull request merged**